.github/workflows/ci-cd.yml

```diff
@@ -71,11 +71,15 @@ jobs:
           sudo apt-get install rpm
           sudo apt-get install snapd
           sudo apt-get install libgpgme-dev libassuan-dev libbtrfs-dev libdevmapper-dev pkg-config
-          git clone https://github.com/containers/skopeo -b v1.6.1 $GITHUB_WORKSPACE/src/github.com/containers/skopeo
-          cd $GITHUB_WORKSPACE/src/github.com/containers/skopeo && make bin/skopeo
+          git clone https://github.com/containers/skopeo -b v1.12.0 $GITHUB_WORKSPACE/src/github.com/containers/skopeo
```

Comment:
Can you also test this is the skopeo used, maybe a `skopeo -v`? The default skopeo (1.4, shipped with the runner, is probably also somewhere in `$PATH`)

The list of software is here: https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md